dashboard/assets/packages/ui/src/App/App.css.ts

```diff
@@ -14,3 +14,7 @@ export const container = style({
 export const main = style({
   padding: vars.sizes.size6
 })
+
+globalStyle('.u-legend', {
+  height: '55px',
+})
```

Comment:
Since the legends are now dynamic on hover, it's possible that it breaks into tw lines, so having more height here enables support for smaller windows without pushing the content on the page.

https://github.com/user-attachments/assets/69a657aa-081a-4481-b016-9b08ec1881ac